### ISSUE TITLE
Module 1.1 Assignment

### ISSUE THREAD

**Issue by tarraschk** [Campus-Advisors/campus-advisor-training-tarraschk#1]
Given what you have learned about Git, why would you want to use Git in your classroom?

**Comment by tarraschk** [comment]
Etude du process Campus Advisors pour Sciences Po

**Comment by arelia** [comment]
👋🏾  Hi @tarraschk, can you tell us more about how you plan to use Git and GitHub in your course? You mentioned in https://github.com/Campus-Advisors/campus-advisor-training-tarraschk/issues/10 that your students would like to communicate with technical workers. What are ways you might introduce version control to your students?

**Comment by tarraschk** [comment]
Hello @arelia 

I plan to use GitHub to show to my students how developers really need to work together in order to not "reinvent the wheel".

I will not teach them how to use Git in details, as it would be too hard for them. However, I already explain to them how Git is really important in a real professional environment in order to merge multiple inputs from different people working on the same files.

Also, my students indeed have a more social oriented academical path, and I teach them how to code so that they can communicate a little with an IT team in a professional environment.

So to sum up:
- I introduce git a little just to tell them that it exists and it's used to avoid conflicts between multiple developers
- I show them GitHub in order to introduce "third-party libraries" and "developers communities"
- I also show them GitHub so that they can find good Ruby Gems on GitHub
- I also tell them that when they will join a professional environment, they should expect the developers to talk about Git / GitHub and version control
